store/src/commonMain/kotlin/com/dropbox/external/store5/Fetcher.kt

```diff
@@ -0,0 +1,16 @@
+package com.dropbox.external.store5
+
+import com.dropbox.external.store5.definition.Converter
+import com.dropbox.external.store5.definition.GetRequest
+import com.dropbox.external.store5.definition.PostRequest
+
+
+/**
+ * Gets from remote data source.
+ * @see [Reader]
+ */
+data class Fetcher<Key : Any, Input : Any, Output : Any>(
+    val get: GetRequest<Key, Output>,
```

Comment:
Verbiage feels inconsistent. If it's a converter we should have a getter/poster as well

Reply:
I renamed using Market/Network prefix here https://github.com/MobileNativeFoundation/Store/pull/450/commits/82ddada9c86264d667e89207091860ddf38ef0ae. For `NetworkFetcher` and `NetworkUpdater` I initially modeled `get` and `post` on Ktor's [HttpClient](https://ktor.io/docs/request.html#url). Linked this discussion to #451 so we can revisit later.